### PR TITLE
refactor: font size tweaks for alert component

### PR DESCRIPTION
**Pull request by Highjhacker**
## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

File: resources/views/alert.blade.php [hunk 1]
```diff
@@ -1,3 +1,10 @@
+@props([
+    'message' => null,
+    'messageClass' => 'text-sm',
+    'type' => null,
+    'large' => false,
+])
+
 <div class="alert-wrapper alert-{{ $type }}">
     <div class="alert-icon-wrapper alert-{{ $type }}-icon @if($large ?? false) alert-icon-large @endif">
```
Comment (ItsANameToo) [inline resources/views/alert.blade.php:9]: when adding props, you should remove all the obsolete `?? <default>` constructions

**Review by ItsANameToo — CHANGES_REQUESTED** [review]
is this in line with the design? as defaulting to `text-sm` will mean all existing alerts will now have a small text unless specified otherwise

**Comment by Highjhacker** [comment]
Assumed it would be, but better to just set the default to nothing just in case, and use `text-sm` where I need it when calling that component. Updated @ItsANameToo

**Comment by ItsANameToo** [comment]
@Highjhacker and how is that messageClass going to be used now that you removed it?

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**